app/api/signup_count/[constituency]/route.ts

```diff
@@ -0,0 +1,48 @@
+import { NextRequest, NextResponse } from "next/server";
+import { kv } from "@vercel/kv";
+import { constituencySlugs } from "@/data/constituencySlugs";
+
+export const runtime = "nodejs";
+
+export async function GET(
+  request: NextRequest,
+  { params }: { params: { constituency: string } },
+) {
+  if (constituencySlugs.includes(params.constituency)) {
```

Comment:
Could just do this as a DB read on our embedded SQLite DB, but will almost certainly be slower than the hard-coded approach...

Wonder if we generate a JSON of _just_ the slugs and read that in here (rather than the slugs exported as a js array). Parsing a small JSON _shouldn't_ be noticeably slower, but means that any changes to the slugs are automatically handled by our normal Github action which loads data from the spreadsheet :thinking:

Reply:
Yeah that's probably a better solution. Wasn't totally sure how though!